.github/workflows/ci.yml

```diff
@@ -5,12 +5,27 @@ on:
     branches:
     - master
   pull_request:
+env:
+  GO111MODULE: on
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
```

Comment:
cache operation seems to be duplicated. therefore, I thought it was unnecessary.

https://github.com/golangci/golangci-lint-action#internals